### PR TITLE
thank Chromatic QA in our README

### LINKED ISSUE #2116: add ChromaticQA to our README
Here is a sample they sent as a possible way to acknowledge the support they are giving our open source project:

https://github.com/govuk-react/govuk-react#acknowledgements

### PR DESCRIPTION
**Pull request by cellog**
# Description

This adds a thank you to Chromatic in our README in the format suggested by @domyen

# Issues

Fixes #2116 

# Checklist:

- [X] 1 PR, 1 purpose: my Pull Request applies to a single purpose
  - [X] This PR only contains configuration changes (package.json, etc.)
  - [ ] This PR only contains code changes (if configuration changes are required, do a separate PR first, then re-base)
- [ ] My code follows the style guidelines of this project, including naming conventions
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] If my code adds or changes components, I have written corresponding stories with Storybook
- [ ] I have performed a self-review of my own code
- [ ] If my code involves visual changes, I am adding applicable screenshots to this thread

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -72,6 +72,10 @@ docker-compose -f docker/docker-compose.ci.yml up --abort-on-container-exit
 
 ## Thank you
 
-[<img src="https://user-images.githubusercontent.com/624104/52508260-d0daa180-2ba8-11e9-970c-3ef9596f6b4e.png" height="35">](https://browserstack.com)
+<a href="https://www.chromaticqa.com/"><img src="https://cdn-images-1.medium.com/letterbox/147/36/50/50/1*oHHjTjInDOBxIuYHDY2gFA.png?source=logoAvatar-d7276495b101---37816ec27d7a" alt="Chromatic Logo" width="120"/></a>
 
-Thank you to BrowserStack for their open source browser testing infrastructure.
+Thanks to [Chromatic](https://www.chromaticqa.com/) for providing the visual testing platform that help us catch unexpected changes on time.
+
+<a href="https://www.browserstack.com/"><img src="https://live.browserstack.com/images/opensource/browserstack-logo.svg" alt="BrowserStack Logo" width="120"> </a>
```
Comment (julien51) [inline README.md]: Any reason you changed that from markdown to HTML?

**Comment by cellog** [comment]
I copy pasted from the Auth0 readme that the Chromatic person recommended, I did not see that it was different, and I'll change it back to markdown for consistency

File: README.md [hunk 2]
```diff
@@ -72,6 +72,10 @@ docker-compose -f docker/docker-compose.ci.yml up --abort-on-container-exit
 
 ## Thank you
 
-[<img src="https://user-images.githubusercontent.com/624104/52508260-d0daa180-2ba8-11e9-970c-3ef9596f6b4e.png" height="35">](https://browserstack.com)
+[<img src="https://cdn-images-1.medium.com/letterbox/147/36/50/50/1*oHHjTjInDOBxIuYHDY2gFA.png?source=logoAvatar-d7276495b101---37816ec27d7a" alt="Chromatic Logo" width="120"/>](https://www.chromaticqa.com/)
 
-Thank you to BrowserStack for their open source browser testing infrastructure.
+Thanks to [Chromatic](https://www.chromaticqa.com/) for providing the visual testing platform that help us catch unexpected changes on time.
+
+[<img src="https://live.browserstack.com/images/opensource/browserstack-logo.svg" alt="BrowserStack Logo" width="120">](https://www.browserstack.com/)
```
Comment (julien51) [inline README.md]: Any reason why you changed the image URL?

**Comment by cellog** [comment]
Here is the README for this PR:

https://github.com/unlock-protocol/unlock/blob/greg-2116-thanks/README.md

Here is the README in master:

https://github.com/unlock-protocol/unlock/blob/master/README.md

I think the one in master looks better, personally, so I'm going to use the existing image.

**Comment by cellog** [comment]
update, PTAL @julien51 

New file can be viewed here: https://github.com/unlock-protocol/unlock/blob/cee7a1f0741bedf28e20ec66cc8001fae6705fb9/README.md

**Pull request merged**